Formula/cclive.rb

```diff
@@ -14,16 +13,58 @@ class Cclive < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "quvi"
   depends_on "boost"
+  depends_on "glibmm"
   depends_on "pcre"
+  depends_on "libquvi"
 
   conflicts_with "clozure-cl", :because => "both install a ccl binary"
 
+  # Upstream PR from 26 Dec 2014 "Add explicit <iostream> includes, fixes build
+  # with Boost 1.56"
+  patch do
+    url "https://github.com/legatvs/cclive/pull/2.patch?full_index=1"
+    sha256 "a4cc99e6b78701c8106871b690c899b95d36d8f873ff4d212e63d8f3f45a990f"
+  end
+
+  # Fix build errors due to assumption of glibc's strerror_r and due to C++11
+  # requirement that there be a space between literal and identifier.
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/1266537/cclive/cxx11-and-strerror_r.diff"
+    sha256 "38ce495646de295e8cb2d6712d82f2d995db0601649197bc17ab01c5027e7845"
+  end
+
+  resource "luasocket" do
+    url "https://luarocks.org/luasocket-3.0rc1-2.src.rock"
```

Comment:
Someone really needs to have a word with Luarocks about redirects someday:
```
x-object-url: http://commondatastorage.googleapis.com/moonrocks/54/luasocket-3.0rc1-2.src.rock?1
```
That URL supports `https://` but sure, whatever 😓.